chrome/content/zotero/zoteroPane.xul

```diff
@@ -106,7 +106,19 @@
 					<toolbarbutton id="zotero-tb-library-add-menu" class="zotero-tb-button" tooltiptext="&zotero.toolbar.newLibrary.label;" type="menu">
 						<menupopup id="zotero-tb-library-add-popup">
 							<menuitem id="zotero-tb-group-add" label="&zotero.toolbar.newGroup;" oncommand="ZoteroPane_Local.newGroup()"/>
-							<menuitem id="zotero-tb-feed-add" label="&zotero.toolbar.newFeed.label;" command="cmd_zotero_newFeed"/>
+							<menu id="zotero-tb-feed-add-menu" label="&zotero.toolbar.feeds.new;">
```

Comment:
I think the options are 1) no submenu, just "New Feed…" and "New Feed from Page…" top-level items, with the latter not appearing in Standalone, or 2) separate "Discover…", "From URL…", and "From Page…" options in the submenu, with the latter not appearing in Standalone.

I guess, in the interest of expediency, we can go with option 2.